src/components/SubHeader/index.tsx

```diff
@@ -3,18 +3,24 @@ import { palette } from "@leafygreen-ui/palette";
 import { Body } from "@leafygreen-ui/typography";
 import Icon from "components/Icon";
 import { StyledLink } from "components/styles";
+import { getEvergreenTaskURL } from "constants/externalURLTemplates";
 import { fontSize, size, subheaderHeight } from "constants/tokens";
 import { useLogContext } from "context/LogContext";
 
 const { gray } = palette;
 
-/** TODO: EVG-17534 */
 interface SubHeaderProps {
   isUploadedLog: boolean;
 }
 const SubHeader: React.FC<SubHeaderProps> = ({ isUploadedLog }) => {
   const { logMetadata } = useLogContext();
-  const { fileName } = logMetadata || {};
+  const { fileName, taskID, execution } = logMetadata || {};
+  let taskLink = "";
+
+  if (taskID !== undefined && execution !== undefined) {
+    taskLink = getEvergreenTaskURL(taskID, execution);
+  }
```

Comment:
just a suggestion, it might be a matter of preference
```suggestion
  const taskLink =
    taskID !== undefined && execution !== undefined
      ? getEvergreenTaskURL(taskID, execution)
      : "";
```

Reply:
Your preference is better :)